src/Components/Navbar.tsx

```diff
@@ -27,7 +27,6 @@ const Navbar = () => {
 
   const handleLogout = () => {
     localStorage.removeItem('token');
```

Comment:
How is this fixing the issue of login?

Reply:
Sir, After session timeout it is navigating from home page to login page showing alert only once and it is not showing the alert again. And I thought the issue is fixed because of removing the navigate('/login'). And opened a MR for it thinking that the issue is solved.

But just now I tested it again and I got alert that "session is expired" at login.

I realized that the issue is not fixed. Just give me some time sir, I will try to find a solution for this issue, by understanding the reasons for this issue

Reply:
ok